feature-detects/mediaquery/hovermq.js

```diff
@@ -12,5 +12,5 @@
 Detect support for Hover based media queries
 */
 define(['Modernizr', 'addTest', 'mq'], function(Modernizr, addTest, mq) {
-  Modernizr.addTest('hovermq', mq(('(hover)')));
```

Comment:
why this change?

Reply:
I just saw the extra parentheses while I was going through similar patterns. I can remove it if needed, but I thought a bit of cleanup never hurts. Let me know if you want to remove that commit from this PR and submit it separately.

Reply:
no we're good, just making sure there wasn't an issue